src/aleph/vm/hypervisors/firecracker/microvm.py

```diff
@@ -95,9 +94,9 @@ def __str__(self):
         return f"vm-{self.vm_id}"
 
     @property
-    def namespace_path(self):
+    def namespace_path(self) -> str:
         firecracker_bin_name = os.path.basename(self.firecracker_bin_path)
-        return f"{JAILER_BASE_DIRECTORY}/{firecracker_bin_name}/{self.vm_id}"
+        return str(self.jailer_base_directory / firecracker_bin_name / str(self.vm_id))
```

Comment:
Why `str()` instead of `.name` ?

```suggestion
        return (self.jailer_base_directory / firecracker_bin_name / str(self.vm_id)).name
```